compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/XCLBinGen.cpp

```diff
@@ -686,6 +685,8 @@ static LogicalResult generateUnifiedObject(MLIRContext *context,
 
   pm.addNestedPass<AIE::DeviceOp>(
       mlir::iree_compiler::AMDAIE::createAIELocalizeLocksPass());
+  pm.addNestedPass<AIE::DeviceOp>(
+      xilinx::AIE::createAIENormalizeAddressSpacesPass());
```

Comment:
Can you remove this - I'll readd when I bring over `AIENormalizeAddressSpacesPass`.

Reply:
That will fail CI,

Reply:
done (by moving elsewehre)